File: src/main/java/postBox/Box3.java
package postBox;

public class Box3 extends abstractBox {

    public Box3(Float length, Float width, int tall) {
        super(length, width, tall);
    }

    @Override
    public boolean validate(float length, float width, int tall) {
        if(check(length, width, tall)) {
            System.out.println("Box3");
            return true;
        }
        return false;
    }
}

File: src/main/kotlin/postBoxK/Box3.kt
package postBoxK

class Box3(length: Float, width: Float, tall: Int) : abstractBox(length, width, tall){

    override fun validate(length: Float, width: Float, tall: Int): Boolean =
        if(check(length, width, tall)){
            println("Box3")
            true
        }else
            false

}